app.rb

```diff
@@ -14,6 +15,13 @@ def initialize()
     @rentals = []
   end
```

Comment:
- [ ] On startup the app loads all data from files in the `initialize` method. Let's fix that
Ex: `@books = load_books()`

Reply:
@EricMbouwe . I'm I supposed to get rid of the instance variable array from the initialize method ? :thinking: :thought_balloon: